osu.Game/Screens/Play/HUD/UnstableRateCounter.cs

```diff
@@ -67,65 +48,18 @@ private void updateDisplay()
 
             double? unstableRate = unstableRateResult?.Result;
 
-            valid.Value = unstableRate != null;
+            IsValid = unstableRate != null;
 
             if (unstableRate != null)
                 Current.Value = (int)Math.Round(unstableRate.Value);
         }
 
-        protected override IHasText CreateText() => new TextComponent
-        {
-            Alpha = alpha_when_invalid,
-        };
-
         protected override void Dispose(bool isDisposing)
         {
             base.Dispose(isDisposing);
 
-            if (scoreProcessor.IsNotNull())
-            {
```

Comment:
please restore this check, it was there for a reason (ensures that this component is safe to dispose before dependency injection has taken place), it cannot be just removed like this for no reason